.github/workflows/release.yaml

```diff
@@ -5,7 +5,32 @@ on:
       - release
 
 jobs:
+  create-draft-release:
```

Comment:
I think this can also be achieved via the electron builder configuration: https://www.electron.build/configuration/publish

Reply:
To be honest I couldn't find it here, I just wanted to automate 1 and 3 steps from:

```
Recommended GitHub Releases Workflow[¶](https://www.electron.build/configuration/publish#recommended-github-releases-workflow)
[Draft a new release](https://help.github.com/articles/creating-releases/). Set the “Tag version” to the value of version in your application package.json, and prefix it with v. “Release title” can be anything you want.

For example, if your application package.json version is 1.0, your draft’s “Tag version” would be v1.0.

Push some commits. Every CI build will update the artifacts attached to this draft.

Once you are done, publish the release. GitHub will tag the latest commit for you.
```